classes/models/submission.php

```diff
@@ -343,20 +343,20 @@ public function submit_plagiarism($type = null) {
         $files = $fs->get_area_files($this->get_context_id(), 'mod_coursework', 'submission',
             $this->id, "id", false);
 
-        $params = array(
+        $params = [
             'context' => \context_module::instance($this->get_coursework()->get_course_module()->id),
             'courseid' => $this->get_course_id(),
             'objectid' => $this->id,
             'relateduserid' => $this->get_author_id(),
             'userid' => $USER->id,
-            'other' => array(
+            'other' => [
                 'content' => '',
-                'pathnamehashes' => array_keys($files)
-            )
-        );
+                'pathnamehashes' => array_keys($files),
+            ],
+        ];
 
         $event = \mod_coursework\event\assessable_uploaded::create($params);
-        //$event->set_legacy_files($files);
+        // $event->set_legacy_files($files);
```

Comment:
really shouldn't be using legacy files or legacy logs at this point

Reply:
Removed in commit 6669562